app/statistics/server/lib/statistics.ts

```diff
@@ -376,7 +573,7 @@ export const statistics = {
 						totalOutgoingActive: integrations.filter(
 							(integration) => integration.enabled === true && integration.type === 'webhook-outgoing',
 						).length,
-						totalWithScriptEnabled: integrations.filter((integration) => integration.scriptEnabled === true).length,
+						totalWithScriptEnabled: integrations.filter((integration) => (integration as any).scriptEnabled === true).length,
```

Comment:
There's `IIntegration` type for this 👀 you'll only need to update the type and add `scriptEnabled: boolean` to the current definition